<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeliculasPrem</class>
 <widget class="QDialog" name="PeliculasPrem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="peli4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="peli2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Alquiler y Compra de Peliculas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="peli3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="peli1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
